inspirehep/modules/workflows/config.py

```diff
@@ -186,3 +186,24 @@
 """Configuration for matching records JCAP and JHEP records since
 they have a particular configuration and we have to look at the
 year as well for accurate matching."""
+
+WORKFLOWS_REFERENCE_MATCHER_DATA_CONFIG = {
```

Comment:
Probably `inspirehep/modules/workflows/config.py` is not the most appropriate place for this guy :) 
There are already similar configurations in https://github.com/inspirehep/inspire-next/blob/master/inspirehep/config.py, like `EXACT_MATCH` and `FUZZY_MATCH`

Reply:
I think module specific configurations would be a better way to do this, no?

Reply:
We discussed this in the team and we believe that having specific config files per module is a better solution than having a big centralized config file.

Reply:
I prefer this way as well! My suggestion was just to keep using the same style around the peoject, but if we agree on this we can start moving configs in related modules.